prow/spyglass/podlogartifact_test.go

```diff
@@ -375,3 +376,79 @@ func TestReadAll_PodLog(t *testing.T) {
 	}
 
 }
+
+type fakeAgent struct {
+	contents string
+}
+
+func (f *fakeAgent) GetProwJob(job, id string) (prowapi.ProwJob, error) {
+	return prowapi.ProwJob{}, nil
+}
+
+func (f *fakeAgent) GetJobLog(job, id, container string) ([]byte, error) {
+	return []byte(f.contents), nil
+}
+
+func TestPodLogArtifact_RespectsSizeLimit(t *testing.T) {
```

Comment:
Some positive testcases would probably be good just to validate that we are not always erring out

Reply:
Done.